src/Renci.SshNet/ScpClient.NET.cs

```diff
@@ -268,5 +283,11 @@ private void InternalDownload(IChannelSession channel, Stream input, FileSystemI
                 SendConfirmation(channel, 1, string.Format("\"{0}\" is not valid protocol message.", message));
             }
         }
+
+        private static string SanitizeUnixFilenameForWindows(string fileName)
+        {
+            fileName = Path.GetInvalidFileNameChars().Aggregate(fileName, (current, c) => current.Replace(c.ToString(), string.Empty));
```

Comment:
I'm not sure If I should seek treatment for this, but I'm beginning to get aversion to Linq.
Also please log a message using ```DiagnosticAbstraction.Log(string)``` when at least one character needs to be "sanitized".